launch_scripts/launch.sh

```diff
@@ -1,22 +1,22 @@
-#!/bin/bash
-
-# Setup
-echo "Setting up..."
-cd /workspaces/REAPER/src/
-source install/setup.bash
-
-# Running
-echo "Launching nodes..."
-ros2 launch controls launch.yaml &
-ros2 launch teleop launch.yaml &
-ros2 run joy joy_node
-
-# Teardown
-echo "Shutting down..."
-pkill ros2
-pkill dig
-pkill dump
-pkill distrib
-pkill drivetrain
-pkill uart
-echo "Killed background processes"
+#!/bin/bash
+
```

Comment:
I agree with the idea to add a build flag, but i don't think it is required for this particular branch to merge.